Mathlib/Order/OrderType.lean

```diff
@@ -0,0 +1,450 @@
+/-
+Copyright (c) _
+-/
+import Mathlib.SetTheory.Cardinal.Basic
+import Mathlib.SetTheory.Ordinal.Basic
+import Mathlib.Order.Category.LinOrd
```

Comment:
I'm very suspicious of anything outside of category theory (or bespoke algebra beyond my mortal understanding) importing category theory. But maybe there's a good reason; care to explain?